Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -554,6 +554,19 @@ noncomputable def algEquiv (K : Type*) [CommRing K] [Algebra A K] [IsFractionRin
     FractionRing A ≃ₐ[A] K :=
   Localization.algEquiv (nonZeroDivisors A) K
 
+theorem algEquiv_comp_algebraMap (K B L : Type*) [CommRing K] [Algebra A K] [CommRing B]
+    [IsDomain B] [CommRing L] [Algebra B L] [IsFractionRing A K] [IsFractionRing B L] [Algebra A B]
+    [Algebra (FractionRing A) (FractionRing B)] [Algebra K L] [Algebra A L] [IsScalarTower A K L]
+    [IsScalarTower A (FractionRing A) (FractionRing B)] [IsScalarTower A B (FractionRing B)]
+    [IsScalarTower A B L] :
+    (algEquiv B L : _ →+* L).comp (algebraMap (FractionRing A) (FractionRing B)) =
+      (algebraMap K L).comp (algEquiv A K) := by
+  let _ : Field L := IsFractionRing.toField B
+  refine IsFractionRing.ringHom_ext (A := A) (fun x ↦ ?_)
+  simp only [RingHom.coe_comp, RingHom.coe_coe, Function.comp_apply, AlgEquiv.commutes]
+  rw [← IsScalarTower.algebraMap_apply, ← IsScalarTower.algebraMap_apply,
+    ← AlgEquiv.restrictScalars_apply A, AlgEquiv.commutes]
```

Comment:
This is already true for `Localization.algEquiv` right? Can you also add that? And an applied version might be useful as well.

Reply:
I've added `IsLocalization.algEquiv_comp_algebraMap` that is as general as possible (I think). I've also included an applied version.